packages/plugins/operators/operators-js/src/operators/shared/array.js

```diff
@@ -76,7 +76,7 @@ const meta = {
     prep,
     validTypes: ['array', 'object'],
   },
-  reverse: { prep, validTypes: ['array'], singleArg: true },
+  reverse: { prep, validTypes: ['array', 'null'], singleArg: true },
```

Comment:
Why are we only changing reverse and not filter, map, etc.. And if null is the input, should and array be the output or simply `null`?